docs/examples/junit5/redis/build.gradle

```diff
@@ -0,0 +1,12 @@
+description = "Examples for docs"
+
+dependencies {
+    compile "io.lettuce:lettuce-core:5.1.1.RELEASE"
+
+    def junitJupiterVersion = '5.4.2'
+    testCompile "org.junit.jupiter:junit-jupiter-api:$junitJupiterVersion"
```

Comment:
Please could you inline the versions and remove the variable? Even though it causes duplication, Dependabot can then keep the versions up to date.

Reply:
done